e2e/dractions/crud.go

```diff
@@ -188,9 +188,15 @@ func createPlacementManagedByRamen(ctx types.TestContext, name, namespace string
 	return nil
 }
 
-func generateDRPCDiscoveredApps(name, namespace, clusterName, drPolicyName, placementName,
-	appname, protectedNamespace string,
+func generateDRPCDiscoveredApps(clusterName, drPolicyName,
+	appname string, ctx types.TestContext,
+	recipeRef *ramen.RecipeRef,
 ) *ramen.DRPlacementControl {
+	name := ctx.Name()
+	namespace := ctx.ManagementNamespace()
+	placementName := name
+	protectedNamespace := ctx.AppNamespace()
```

Comment:
Why do need these? they are not use in the function. It seems the only change is adding the recipeRef.